cpg_workflows/jobs/mito.py

```diff
@@ -0,0 +1,327 @@
+"""
+Hail Batch jobs needed to call mitochondrial SNVs
+"""
+import hailtop.batch as hb
+from hailtop.batch.job import Job
+
+from cpg_utils import Path, to_path
+from cpg_utils.hail_batch import image_path, fasta_res_group
+from cpg_utils.hail_batch import command
+from cpg_workflows.resources import STANDARD
+from cpg_workflows.filetypes import CramPath
+
+
+def subset_cram_to_chrM(
+    b,
+    cram_path: CramPath,
+    job_attrs: dict | None = None,
+) -> Job:
+    """
+    Extract read pairs with at least one read mapping to chrM
+
+    Specific config for selection of which reads to extract may influence the impact of
+    NUMTs. Because of this we stick to the exact command used by Broad until we have a
+    good reason not to.
+
+    Args:
+        cram_path: Input cram to extract chrM reads from.
+
+    Outputs:
+        job.output_bam: a ResourceGroup containing bam of reads mapped to chrM
+
+    Cmd from:
+    https://github.com/broadinstitute/gatk/blob/330c59a5bcda6a837a545afd2d453361f373fae3/scripts/mitochondria_m2_wdl/MitochondriaPipeline.wdl#LL188-L244C2
+
+    """
+
+    job_attrs = job_attrs or {}
```

Comment:
It might be worth including additional job attributes. By default, the dataset and sequencing group id will be added here. This allows us to filter on these params in hail batch, and you or @EddieLF might have other use cases in mind that are more helpful. 
```suggestion
    job_attrs = (job_attrs or {}) | dict(tool='gatk_PrintReads')
```